server/src/security/accessTokenService.ts

```diff
@@ -171,17 +174,28 @@ export function getAccessTokenScope<Schema extends SchemaWithSecurity>(
   )
 }
 
-export function parseAccessToken<Schema extends SchemaWithSecurity>(
+export async function parseAccessToken<Schema extends SchemaWithSecurity>(
   accessToken: string,
   schema: Schema,
   params: PathParam | undefined,
   querystring: QueryString | undefined
-): IAccessToken<Schema> {
+): Promise<IAccessToken<Schema>> {
   const data = jwt.verify(accessToken, config.auth.user.jwtSecret, {
     complete: true,
     issuer: config.publicUrl,
   })
   const token = data.payload as IAccessToken<Schema>
+  if (token.identity.type === "IUserRecruteur") {
+    const user = await getUser({ _id: token.identity._id })
+
+    if (!user) throw Boom.unauthorized()
+
+    const userStatus = controlUserState(user?.status)
+
+    if (userStatus.error) {
+      throw Boom.forbidden("Not Authorized")
```

Comment:
```suggestion
      throw Boom.forbidden()
```
Le "Not authorized" prête à confusion avec une 401.